toolchain/check/testdata/interop/cpp/no_prelude/function_decl.carbon

```diff
@@ -154,7 +154,7 @@ fn F() {
 
 // --- non_void_return_function_decl.h
 
-int foo();
+bool foo();
```

Comment:
I suggest changing this to a `struct` so even when we support this it can stay in `no_prelude`. I think all primitives will require prelude.